helpers/cert-manager/.helmignore

```diff
@@ -21,5 +21,3 @@
 .idea/
 *.tmproj
 .vscode/
-*.crt
-*.key
```

Comment:
scripts helpers/Mongodb-* create crt/key files for the _**cert-manager**_ chart and the chart ignore them